GDJS/Runtime/pixi-renderers/runtimescene-pixi-renderer.ts

```diff
@@ -120,47 +149,214 @@ namespace gdjs {
           if (!cameraCoords || !rendererObject) {
             continue;
           }
 
+          // Create the structure to store the points in memory
+          if (!this._debugDrawRenderedObjectsPoints[id]) {
+            this._debugDrawRenderedObjectsPoints[id] = {
+              wasRendered: true,
+              points: {},
+            };
+          }
+
+          const renderedObjectTextPoints = this._debugDrawRenderedObjectsPoints[
+            id
+          ];
+
           // Draw hitboxes (sub-optimal performance)
           const hitboxes = object.getHitBoxes();
           for (let j = 0; j < hitboxes.length; j++) {
             // Note that this conversion is sub-optimal, but we don't care
             // as this is for debug draw.
             const polygon: float[] = [];
             hitboxes[j].vertices.forEach((point) => {
-              polygon.push(point[0]);
-              polygon.push(point[1]);
+              if (layer) {
+                polygon.push(point[0] - cameraTopLeftX);
+                polygon.push(point[1] - cameraTopLeftY);
+              }
             });
             debugDraw.fill.alpha = 0;
-            debugDraw.line.color = 0xe86868;
+            debugDraw.line.alpha = 0.5;
+            debugDraw.line.color = 0xff0000;
             debugDraw.drawPolygon(polygon);
           }
 
-          // Draw circle point
-          debugDraw.fill.alpha = 0.8;
-          debugDraw.line.color = 0x68e868;
-          debugDraw.fill.color = 0x68e868;
-          debugDraw.drawCircle(object.getDrawableX(), object.getDrawableY(), 3);
-
-          // Draw center point
-          debugDraw.fill.alpha = 0.8;
-          debugDraw.line.color = 0xe8e868;
-          debugDraw.fill.color = 0xe8e868;
-          debugDraw.drawCircle(
-            object.getDrawableX() + object.getCenterX(),
-            object.getDrawableY() + object.getCenterY(),
-            3
-          );
+          // Draw points
+          // Center point of the object, with camera movement
+          const centerPointX =
+            object.getDrawableX() + object.getCenterX() - cameraTopLeftX;
+          const centerPointY =
+            object.getDrawableY() + object.getCenterY() - cameraTopLeftY;
+
+          // Draw Center point
+          debugDraw.fill.alpha = 0.3;
+          debugDraw.line.color = 0xffff00;
+          debugDraw.fill.color = 0xffff00;
+          debugDraw.drawCircle(centerPointX, centerPointY, 3);
+
+          if (showPointsNames) {
+            if (!renderedObjectTextPoints.points['center']) {
+              renderedObjectTextPoints.points['center'] = new PIXI.Text(
+                'Center',
+                {
+                  fill: '#ffff00',
+                  fontSize: 12,
+                }
+              );
+            }
+
+            renderedObjectTextPoints.points['center'].position.set(
+              centerPointX,
+              centerPointY
+            );
+
+            this._pixiContainer.addChild(
```

Comment:
addChild is being called at each render, which is useless and will impact performance. I'm fixing this.